src/configs.rs

```diff
@@ -15,11 +18,60 @@ pub(crate) struct Opts {
 #[derive(Clap, Debug)]
 pub(crate) enum SubCommand {
     /// Run NEAR Indexer Example. Start observe the network
-    Run,
+    Run(RunArgs),
     /// Initialize necessary configs
     Init(InitConfigArgs),
 }
 
+#[derive(Clap, Debug)]
+pub(crate) struct RunArgs {
+    /// streamer SyncMode. Possible values: from-interruption, last-synced and block. Default: from-interruption
+    #[clap(long, default_value = "from-interruption")]
+    pub sync_mode: SyncMode,
```

Comment:
It is fine to have it this way, but if you want to learn how we can improve it, consider using sync modes as separate subcommands where only SyncFromBlock subcommand will have the block height (and it will be a required/non-Option field)